10up/distributor
# Add view all post types filter to External Connections.

### Description of the Change


Introduce a "View All" post type view to the pull screen for external connections for parity with the internal connections. The new view is the default when visiting Distributor > Pull Content > [External connection].

Closes #861 

### How to test the Change


1. Create two separate sites using insta wp.
2. Install distributor plugin on both the sites. Connect the sites.
3. Create a custom post type "Book" on both the sites.
4. On pull screen ensure that "View All" option is available.
5. Pull content from CPT and it should pull the posts from external sites.

### Changelog Entry


> Added - Introduce "View all" post type view when pulling content from external connections.



### Credits

Props @ankitrox, @Sidsector9, @ravinderk, @jeffpaul, @peterwilsoncc 


### Checklist:
